financial-connections/src/main/java/com/stripe/android/financialconnections/features/partnerauth/PartnerAuthViewModel.kt

```diff
@@ -58,8 +58,34 @@ internal class PartnerAuthViewModel @Inject constructor(
 
     init {
         logErrors()
-        observePayload()
-        createAuthSession()
+        withState {
+            if (it.firstInit) {
+                setState { copy(firstInit = false) }
+                launchBrowserIfNonOauth()
+                createAuthSession()
+            } else {
+                restoreAuthSession()
+            }
+        }
+    }
```

Comment:
We don't want to create a new session if coming from a process kill. We'll restore the session from the manifest instead.